File: Programa1_operaciones_cadena/Programaa_operaciones/Operaciones.java


public class Operaciones
{
	public static void main(String[] args) {
		
		long max = 100000000;

		double i = 0;
		double seno = 0;
		double coseno = 0;
		double tang = 0;;
		double logaritmo = 0;;
		double raiz = 0;;

		while(i<max)
		{
			seno += Math.sin(i);
			coseno += Math.cos(i);
			tang += Math.tan(i);
			logaritmo += Math.log10(i);
			raiz += Math.sqrt(i);
			i++;
		}

	}
}

File: Programa1_operaciones_cadena/Programaa_operaciones/operaciones.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

void main(){
	double i = 0;
	int max = 100000000;
	double seno = 0;
	double cose = 0;
	double tang = 0;
	double loga = 0;
	double raiz = 0;


	while(i<max){
		seno += sin(i);
		cose += cos(i);
		tang += tan(i);
		loga += log10(i);
		raiz += sqrt(i);
		i++;
	}

}